### DISCUSSION TITLE
Button for input selector?

### DISCUSSION THREAD

**Discussion by darkner258** [Nerwyn/universal-remote-card#46] [Q&A]
Hey there, first of all many thanks for this, it's an awesome piece of work!
My question is, is there a command to open the input selector popup? Or the individual inputs (HDMI 1, HDMI 2, etc.)? 

Thanks in advance!

**Comment by Nerwyn** [comment]
What is the input selector popup? My Google TV has as an input select button but it's just remappable to IR commands (I use button remapper to remap it to Home Assistant service calls). I don't see anything about an input selector on the [Home Assistant Android TV Remote integration](https://www.home-assistant.io/integrations/androidtv_remote/) page either.

**Comment by darkner258** [comment]
I mean this dialogue
![image](https://github.com/Nerwyn/android-tv-card/assets/106254929/7d199ae6-97d2-4bda-9666-5fcc6f63b051)

**Reply by Nerwyn** [reply]
Are you sure that's part of Android TV or is it part of your TV? Does your tv have Android TV built in? How are you bringing up that menu?

**Reply by darkner258** [reply]
It does, I bring it up using the input button on the remote, bottom right grey one 
![image](https://github.com/Nerwyn/android-tv-card/assets/106254929/fc0f98df-8534-4b9d-82f5-5258f056df0d)

**Reply by Nerwyn** [reply]
Yeah it looks like this is a command to your TV and not Android TV, so you won't be able to send it using the Android TV integrations for Home Assistant.

Do you know if it's an IR remote command? Like, does it only work if you're pointing it at your TV? I use a Broadlink IR/RF remote to learn and send IR/RF remote commands, and then wrap those commands in scripts and call them in my remote.

**Comment by darkner258** [comment]
Yeah it is IR, that might be the way to go. I have a switchbot hub with IR
and this button setup. Is there a way to make it into a script in HA?

**Reply by Nerwyn** [reply]
I'm not sure how to setup and learn commands for Switchbot IR with Home Assistant, but once you do that you can make scripts by going to `Settings` > `Automationm & scenes` > `Scripts` > `+ ADD SCRIPT`. From there mine look like this:

```yaml
icon: mdi:volume-high
alias: Lounge TV - Amplifier Volume Up
sequence:
  - service: remote.send_command
    target:
      entity_id: remote.lounge
    data:
      device: amplifier
      command: volume_up
mode: parallel
max: 2
```
